db/table_cache.cc

```diff
@@ -62,14 +62,17 @@ void AppendVarint64(IterKey* key, uint64_t v) {
 
 }  // namespace
 
+#define LOAD_CONCURENCY 128
```

Comment:
Can it be a const instead of a macro? The memory is dynamically allocated anyway.

Reply:
It can. I was waiting for feedback if you want it in options (I would not put it there). And I saw other macros as well. But yes can change it.

Reply:
Fixed